src/main/java/zabi/minecraft/nbttooltip/config/ConfigInstance.java

```diff
@@ -33,15 +33,15 @@ public ConfigInstance() {
 		copyingEngine = CopyingEngine.JSON;
 	}
 	
-	public static enum TooltipEngine {
+	public enum TooltipEngine {
 		
 		FRIENDLY(new ColoredHumanReadableParser()), 
 		FRIENDLY_NO_COLOR(new BWHumanReadableParser()), 
 		JSON(new JsonParser());
 		
 		private NbtTagParser parser;
 		
-		private TooltipEngine(NbtTagParser parser) {
+		TooltipEngine(NbtTagParser parser) {
```

Comment:
default visibility triggers warnings on my IDE, and it is generally better to keep access to a minimum. This line should be reverted

Reply:
IntelliJ was complaining about it on my end, I can revert it as long as it doesn't mess anything up.

Reply:
![image](https://user-images.githubusercontent.com/26803198/121933057-ded6f900-cd45-11eb-8b13-0156eece6af7.png)
This is in java 16, idk if that makes any differences.

Reply:
I guess I can just add an ignore annotation. Keep this change, it's ok